src/xtgeo/cube/_cube_import.py

```diff
@@ -38,12 +39,13 @@
 import segyio
 from segyio import TraceField as TF
 
-import xtgeo
 import xtgeo.common.calc as xcalc
+import xtgeo.common.constants as const
 import xtgeo.common.sys as xsys
 from xtgeo import _cxtgeo
```

Comment:
Technically, these are not relative (?)

Reply:
Yep, but since this is importing stuff in `xtgeo.common` they shouldn't be... would have to do a parent-relative import `import ..common.calc`. If they were in `xtgeo.cube` it could just be `import .something`. I don't think this later point, using relative imports in the same package, is really something _that_ important to maintain everywhere, but it can help reduce circular dependencies by not searching through the whole path.